src/main/resources/templates/modificar-proveedor.html

```diff
@@ -51,24 +51,25 @@ <h3 class="title">EDITAR PROVEEDOR</h3>
                         <option value="Construcción">Construcción</option>
                     </select>
                 </div>
-                   <div class="inputContainer" th:value{$}>
+                <div class="inputContainer" th:value{$}>
```

Comment:
Por favor cuide in crustar estos thymeleaf, para que no generen crashes en el App, si no lo vas a definir, es mejor reportarlo con un Issue para que alguien lo corrija.

Reply:
Corregido